server/backend/cron/cronjobs.go

```diff
@@ -19,7 +19,9 @@ type CronService struct {
 	APNs *apns.Service
 }
 
-const StorageDir = "/Storage/" // target location of files
+// StorageDir is the directory where files are stored
+// this is a variable, so it can be changed during tests
+var StorageDir = "/Storage/" // target location of files
```

Comment:
Why should this not be a const?

Reply:
See the comment above, this is for tests

Reply:
How did I not see that? :see_no_evil: